### ISSUE TITLE
2010 FIFA World Cup Graphical Issue

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#12895]
### Quick summary

When testing this game I came across a graphical  issue not mentioned on the compatible report it seems to only affect 30 series cards

### Details

Rpcs3

https://user-images.githubusercontent.com/71127352/198730372-6d100c90-77d7-4547-8ca2-88688d5fc20b.mp4

Rpcs3

https://user-images.githubusercontent.com/71127352/198730401-be226bd4-a477-4663-8f2f-a6ee64ed4d5e.mp4

Ps3
https://youtu.be/OvyGp1du4xU?t=51

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/9891767/RPCS3.log.gz)

### Attach capture files for visual issues

[NPEB90228_20221029104534_capture.zip](https://github.com/RPCS3/rpcs3/files/9893396/NPEB90228_20221029104534_capture.zip)

### System configuration

- SYS: RPCS3 v0.0.24-14339-c8620070 Alpha | master
- SYS: AMD Ryzen 5 2600 Six-Core Processor | 12 Threads | 15.95 GiB RAM | TSC: 3.400GHz | AVX+ | FMA3
- SYS: Operating system: Windows, Major: 10, Minor: 0, Build: 19044, Service Pack: none, Compatibility mode: 0
- SYS: Current Time: 2022-10-28T21:53:46
- RSX: Found vulkan-compatible GPU: 'NVIDIA GeForce RTX 3060 Ti' running on driver 526.47.0.0
- SYS: Firmware version: 4.89

**Comment by YuriNator557** [comment]
Missing RSX capture

**Comment by Triticum0** [comment]
Added

**Comment by Triticum0** [comment]
Figure out the issue just need to disable vertex cache

**Comment by AniLeo** [comment]
Not a valid fix, needs an actual fix

**Comment by Triticum0** [comment]
This also affects:
FIFA 15 ultimate team
FIFA 16 ultimate team
NHL 12 ultimate team
Crystal Defenders (disabling vertex cache doesn't fix the issue)

Note: NHL 11 ultimate team is not affected also should affect all FIFA games that have an ultimate team after 11 but haven't tested them

**Comment by Triticum0** [comment]
tested with #14880 and didn't make a difference

**Comment by digant73** [comment]
@Triticum0 is that still an issue?

**Comment by Triticum0** [comment]
yes it still an issue as of RPCS3 Version: 0.0.38-18263-3c401e7b Alpha | master
